doc/classes/Array.xml

```diff
@@ -138,18 +127,19 @@
 			<param index="0" name="method" type="Callable" />
 			<description>
 				Calls the provided [Callable] on each element in the array and returns [code]true[/code] if the [Callable] returns [code]true[/code] for [i]all[/i] elements in the array. If the [Callable] returns [code]false[/code] for one array element or more, this method returns [code]false[/code].
-				The callable's method should take one [Variant] parameter (the current array element) and return a boolean value.
+				The [param method] should take one [Variant] parameter (the current array element) and return a boolean value.
 				[codeblock]
```

Comment:
Shouldn't it be `[gdscript]` instead of `[codeblock]` here and in other GDScript examples?

Reply:
Many of Array's methods are not available in C#'s Godot.Collections.Array. At least, not in a way I'm sure can easily be translated.

Reply:
But the codeblock itself should highlight gdscript code. even if this is only showing GDScript.

Reply:
So the C# code example should exist, but either be "This does not exist?" or provide an alternative? Not sure about it.

Maybe @YuriSizov can give more of an opinion. There was a similar argument to be had in https://github.com/godotengine/godot/pull/67880#discussion_r1030505866:
> Also also, it's weird to say "In GDScript" and give examples in both GDScript and C#, with the latter using a different syntax 🙂

Reply:
Nevermind, I thought that it was possible to use `[gdscript]` in a standalone way, without `[codeblocks]`, but it's not possible.

Reply:
Even when the example is provided only for GDScript, I think it's still better to use `[codeblocks][gdscript]` because then it shows the tab that makes it more clear that the code is in GDScript.

About translating examples to C#, the table in the [_C# Collections_](https://docs.godotengine.org/en/stable/tutorials/scripting/c_sharp/c_sharp_collections.html) documentation page should help with that, but let me know if there are any code examples that you are not sure how to translate.

In this example, you can use the Linq [`All`](https://learn.microsoft.com/en-us/dotnet/api/system.linq.enumerable.all) method which is very similar to the GDScript version.

Reply:
Could you provide a similar example to the GDScript one used, for GDScript's arrays?

Reply:
```csharp
private static bool GreaterThan5(int number)
{
	return number > 5;
}

public override void _Ready()
{
	GD.Print(new Godot.Collections.Array<int> { 6, 10, 6 }.All(GreaterThan5)); // Prints true (3/3 elements evaluate to true).
	GD.Print(new Godot.Collections.Array<int> { 4, 10, 4 }.All(GreaterThan5)); // Prints false (1/3 elements evaluate to true).
	GD.Print(new Godot.Collections.Array<int> { 4, 4, 4 }.All(GreaterThan5)); // Prints false (0/3 elements evaluate to true).
	GD.Print(new Godot.Collections.Array<int> { }.All(GreaterThan5)); // Prints true (0/0 elements evaluate to true).

	// Same as the first line above, but using a lambda function.
	GD.Print(new Godot.Collections.Array<int> { 6, 10, 6 }.All(element => element > 5)); // Prints true
}
```